tgui/packages/tgui/interfaces/CrewMonitor.js

```diff
@@ -1,98 +1,145 @@
 import { sortBy } from 'common/collections';
-import { useBackend } from "../backend";
+import { useBackend, useLocalState } from "../backend";
 import { Window } from "../layouts";
-import { NanoMap, Box, Table, Button } from "../components";
+import { NanoMap, Box, Table, Button, Tabs, Icon, NumberInput } from "../components";
 import { TableCell } from '../components/Table';
 
 export const CrewMonitor = (props, context) => {
   const { act, data } = useBackend(context);
+  const [tabIndex, setTabIndex] = useLocalState(context, 'tabIndex', 0);
   const crew = sortBy(
-    crewmember => crewmember.name,
+    cm => cm.name,
   )(data.crewmembers || []);
+  const [
+    mapZoom,
+    setZoom,
+  ] = useLocalState(context, 'number', 1);
+  let body;
+  // Data view
+  if (tabIndex === 0) {
+    body = (
+      <Table>
+        <Table.Row header>
+          <Table.Cell>
+            Name
+          </Table.Cell>
+          <Table.Cell>
+            Status
+          </Table.Cell>
+          <Table.Cell>
+            Location
+          </Table.Cell>
+        </Table.Row>
+        {crew.map(cm => (
+          <Table.Row key={cm.name}>
+            <TableCell>
+              {cm.name} ({cm.assignment})
+            </TableCell>
+            <TableCell>
+              <Box inline
+                color={cm.dead ? 'red' : 'green'}>
+                {cm.dead ? 'Deceased' : 'Living'}
+              </Box>
+              {cm.sensor_type >= 2 ? (
+                <Box inline>
+                  {'('}
+                  <Box inline
+                    color="red">
```

Comment:
For the sake of consistency, you can probably use the damage type colours defined [here](https://github.com/ParadiseSS13/Paradise/blob/58aa86cb9f9db0fcf9abed19b7c22931f69dfabe/tgui/packages/tgui/constants.js#L21) if they a. 
I don't think they're exported (or defined at all) in CSS which is probably something I would do.